test/tools/PSxUnit/Program.cs

```diff
@@ -0,0 +1,226 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Codeblast;
+using PSxUnit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+using Xunit;
+using Xunit.Runners;
+using System.Collections.Concurrent;
+using Xunit.Runner.DotNet;
+using Microsoft.Extensions.DependencyModel;
+using Microsoft.DotNet.InternalAbstractions;
+using Microsoft.DotNet.ProjectModel;
+using System.Linq;
+using System.Xml.Linq;
+using System.IO;
+using System.Xml;
+using System.Management.Automation;
+
+
+public enum TestType
+{
+    CiFact,
+    FeatureFact,
+    ScenarioFact,
+    All
+}
+
+namespace PSxUnit
+{
+
+    public class Options : CommandLineOptions
```

Comment:
Add /// <summary> </summary> style comments for all public types and properties.